### ISSUE TITLE
[a11y] my zlinks - missing select box label

### ISSUE THREAD

**Issue by jxjj** [UMN-LATIS/z#229]
https://z.umn.edu/shortener/urls

Collection select box not labeled.

Audit issue 23.
